superset/commands/importers/v1/examples.py

```diff
@@ -103,6 +104,14 @@ def _import(
                 else:
                     config["database_id"] = database_ids[config["database_uuid"]]
 
+                # set schema
+                if config["schema"] is None:
+                    url = make_url(
+                        conf.get("SQLALCHEMY_EXAMPLES_URI")
+                        or conf.get("SQLALCHEMY_DATABASE_URI")
```

Comment:
```suggestion
                        or conf["SQLALCHEMY_DATABASE_URI"]
```

By construction this is guaranteed to exist.

Also @betodealmeida per [here](https://github.com/apache/superset/blob/master/superset/__init__.py#L36-L39) the `conf` field is deemed to be legacy, i.e., we likely should be using  `current_app.config` instead.

Reply:
Thanks, @john-bodley! I'll update to use `current_app.config` instead.